components/attention/w-attention.vue

```diff
@@ -27,92 +24,139 @@ const props = defineProps({
   ...modelProps({ modelDefault: absentProp }),
   targetEl: Object,
   attentionClass: [Object, String],
-  attentionEl: {
-    default: () => ref(null),
-  },
   role: String,
   ariaLabel: String,
+  placement: {
+    type: String,
+    validator(value) {
+      return [
+        'top-start',
```

Comment:
Just a de-dupe thing but this list could come from the `core` library as well since this was used in Elements as well.

But for now this is ok

Reply:
Ah you're right! I can look into moving it into the `core` library. 👍

Reply:
Updated in this commit: [bda07aa](https://github.com/warp-ds/vue/pull/141/commits/bda07aa92ade74b7131a4992b592a0e6a70efa98)